livre/core/render/TransferFunction1D.h

```diff
@@ -27,105 +28,62 @@
 #include <livre/core/types.h>
 #include <livre/core/mathTypes.h>
 
-#define TF_NCHANNELS 4u
+#include <zerobuf/render/lookupTable1D.h>
 
 namespace livre
 {
 
-/**
- * The TransferFunction1D class holds the color and transparency for an RGBA 1 dimensional Transfer Function ( TF ).
- */
-class TransferFunction1D
+/** Color and transparency for an RGBA 1 dimensional Transfer Function ( TF ). */
+class TransferFunction1D : public ::zerobuf::render::LookupTable1D
 {
-public:
-    /**
-     * Create the transfer function with default parameters.
-     */
-    TransferFunction1D() { reset(); }
+    static const size_t NCHANNELS = 4;
 
-    /**
-     * Create a transfer function.
-     * @param size The number of samples in the transfer function.
-     */
-    explicit TransferFunction1D( const uint32_t size ) { createCustomTF_( size ); }
+public:
+    /** Create the transfer function with default parameters. */
+    LIVRECORE_API TransferFunction1D();
 
     /**
-     * Load transfer function from an ASCII "1dt" file. The content of the file
-     * consists of a first line with the number of sample points in the transfer
-     * function and their format, and then all the values in 'R G B A' format.
-     * Currently both float [0.0f, 1.0f] and 8-bit unsigned integers [0, 255]
-     * values are supported. If the format is unspecified, float is used.
-     * If the file extension or format is not supported or the file could not be
-     * opened, a default transfer function is generated.
+     * Load transfer function.
+     *
+     * Supported are ASCII "1dt", as well as ascii and binary files of
```

Comment:
I don't think GUI can open it. Should we remove this format? Stefan, do you have an opinion?